insolar/logger.go

```diff
@@ -86,8 +86,8 @@ type Logger interface {
 
 	// WithCaller switch on/off 'caller' field computation.
 	WithCaller(flag bool) Logger
-	// WithSkipFrameCount configures skipFrameCount for 'caller' field computation.
-	WithSkipFrameCount(skipFrameCount int) Logger
+	// ChangeSkipFrameCount changes skipFrameCount by delta value.
+	ChangeSkipFrameCount(delta int) Logger
```

Comment:
I don't think this should be renamed. I understand that this method adds or removes delta and it's ok. I prefer "With" prefix, it's more clear that it is a modifier.

Reply:
May be better name would be "WithSkipFrameDelta". What do you think?

Reply:
'delta' is already in the function signature, so my solution is just to stop on WithSkipFrameCount with improved godoc description